pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -136,3 +137,23 @@ func isPrerelease(tag string) bool {
 	}
 	return false
 }
+
+// IsGreaterOrEqualToBaseVersion returns true if the version is greater or equal to the base version
```

Comment:
Why do we actually need to provide the base version as a parameter?

Reply:
@gbraad It basically a compare function of 2 different openshift versions.

Reply:
That is clear, but the naming of the variables confused... this is improved... except for one case.